test/inductor/test_inductor_utils.py

```diff
@@ -0,0 +1,34 @@
+# Owner(s): ["module: inductor"]
+
+import functools
+import logging
+
+import torch
+
+from torch._dynamo.test_case import run_tests, TestCase
+
+from torch._inductor.utils import do_bench, do_bench_using_profiling
+
+
+log = logging.getLogger(__name__)
+
+
+class TestBench(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        x = torch.rand(1024, 10).cuda().half()
+        w = torch.rand(512, 10).cuda().half()
+        cls._bench_fn = functools.partial(torch.nn.functional.linear, x, w)
+
+    def test_do_bench(self):
+        res = do_bench(self._bench_fn)
```

Comment:
tests should assert something

Reply:
There is a check to throw exception inside `do_bench_using_profiling()`, so the test below makes sure that no exception is thrown. I also use this two tests to compare latency collected by these two functions. We could skip the `test_do_bench()` test in CI though...